src/FINDOLOGIC/Export/Data/Attribute.php

```diff
@@ -21,17 +23,18 @@ class Attribute implements Serializable
      */
     public function __construct(string $key, array $values = [])
     {
-        $this->key = DataHelper::checkForEmptyValue($key);
+        $this->key = DataHelper::checkForEmptyValue('attribute', $key);
         $this->setValues($values);
     }
 
     /**
      * @SuppressWarnings(PHPMD.StaticAccess)
+     * @param mixed $value
      */
     public function addValue($value): void
     {
         DataHelper::checkAttributeValueNotExceedingCharacterLimit($this->getKey(), $value);
-        array_push($this->values, DataHelper::checkForEmptyValue($value));
+        array_push($this->values, DataHelper::checkForEmptyValue('attribute', $value));
```

Comment:
Can we implement the `NameAwareValue` class here as well and access the name via the `getValueName()` method instead?